src/FakeClient/FakeClientMain.cpp

```diff
@@ -398,6 +403,7 @@ int main(int argc, char* argv[]) {
     ErrorMessageOr<std::vector<orbit_grpc_protos::ModuleInfo>> modules_or_error =
         orbit_module_utils::ReadModules(options.process_id);
     ORBIT_FAIL_IF(modules_or_error.has_error(), "%s", modules_or_error.error().message());
+    process.UpdateModuleInfos(modules_or_error.value());
```

Comment:
... And then also remove this part? We actually get rid of code duplication this way.